### PR TITLE
Patch the registry-aliases addon missing entrypoint

### LINKED ISSUE #8383: Addon registry-aliases alpine image missing bash command (ash works)
**Steps to reproduce the issue:** 

1. minikube start
2. minikube addons enable registry
3. minikube addons enable registry-aliases

kubelet says:
`
Error: failed to start container "update": Error response from daemon: OCI runtime create failed: container_linux.go:349: starting container process caused "exec: \"bash\": executable file not found in $PATH": unknown
`

Easily fixed by changing the daemonset entrypoint (command[0]) from `bash` to `ash`

### PR DESCRIPTION
**Pull request by tsraza**
Fixes #8383

### CODE REVIEW

**Comment by medyagh** [comment]
@tsraza thanks this PR

File: deploy/addons/registry-aliases/node-etc-hosts-update.tmpl [hunk 1]
```diff
@@ -29,7 +29,7 @@ spec:
                   name: registry-aliases
                   key: registryAliases
           command:
-            - bash
+            - ash
```
Comment (afbjorklund) [inline deploy/addons/registry-aliases/node-etc-hosts-update.tmpl:32]: Could consider using standard /bin/sh here

Comment (medyagh) [reply deploy/addons/registry-aliases/node-etc-hosts-update.tmpl:32]: @tsraza I agree with @afbjorklund

Comment (tsraza) [reply deploy/addons/registry-aliases/node-etc-hosts-update.tmpl]: changed to `sh` and tested locally in virtualbox -- it works

**Pull request merged**